src/main/java/seedu/address/commons/util/DateUtil.java

```diff
@@ -0,0 +1,12 @@
+package seedu.address.commons.util;
+
+import java.time.format.DateTimeFormatter;
+
+/**
+ * A centralized class for Date-Time related functions
+ */
+public class DateUtil {
+
+    public static final DateTimeFormatter DATE_TIME_FORMATTER = DateTimeFormatter.ofPattern("dd-MM-yyyy-HH-mm");
```

Comment:
Good to see general utility items such as these related to datetime being put into a fresh util file!